src/components/Output.jsx

```diff
@@ -76,6 +76,7 @@ class Output extends React.Component {
 
 Output.propTypes = {
   errors: React.PropTypes.object.isRequired,
+  isHidden: React.PropTypes.string,
```

Comment:
From the standpoint of the component’s interface, I would expect this to be a boolean (this will probably occur naturally as part of the change I proposed above, but even independently of that change, passing `"_hidden"` as the `isHidden` prop seems like a leaky implementation detail).